frame/balances/e2e-tests/src/lib.rs

```diff
@@ -0,0 +1,148 @@
+use substrate_test_runner::{Node, ChainInfo};
+use pallet_sudo::Call as SudoCall;
+use pallet_balances::Call as BalancesCall;
+use sp_keyring::sr25519::Keyring::{Alice, Bob};
+use sp_runtime::{traits::IdentifyAccount, MultiSigner, MultiAddress, MultiSignature};
+use sp_runtime::AccountId32;
+use sp_runtime::traits::Block as BlockT;
+use sp_runtime::generic::UncheckedExtrinsic;
+use codec::Encode;
+use frame_support::traits::Get;
+
+type Extrinsic<R, S> = UncheckedExtrinsic<
+    MultiAddress<<R as frame_system::Config>::AccountId, <R as frame_system::Config>::Index>,
+    <R as frame_system::Config>::Call,
+    MultiSignature,
+    S,
+>;
+
+/// Tests the Balances::force_transfer call
+pub fn force_transfer<T>(node: &Node<T>)
+    where
```

Comment:
we talked about just making that a reusable trait alias. Why hasn't that happened?